.github/workflows/ci.yml

```
@@ -2,118 +2,73 @@ name: CI
-    - name: Run CodeQL Analysis
-      uses: github/codeql-action/init@v2
-      with:
-        languages: javascript
-
+      
+    - name: Build project
+      run: npm run build
+      
     - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v2
+      uses: github/codeql-action/analyze@v3
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)